api/conf/settings/base.py

```diff
@@ -40,13 +40,18 @@
 
 ROOT_URLCONF = "conf.urls"
 
+loaders = [
+    "django.template.loaders.filesystem.Loader",
```

Comment:
Do you know if we strictly need the `filesystem.Loader`? `DIRS` is set to just the `api` app's template directory, which makes me think our usage of `filesystem.Loader` is duplicative of `app_directories.Loader`, which already sources templates from `<app_dir>/templates`.  I think we can rely entirely on the `app_directories.Loader`, in that case, right? Maybe we can leave this loader out entirely?